### DISCUSSION TITLE
Как узнать все домены сайта?

### DISCUSSION THREAD

**Discussion by icynstant** [bol-van/zapret#987] [Q&A] [answered]
Я новенький, только начинаю разбираться. Хочу получить доступ к заблокированному сайту, без --hostlist-domains="" всё получается, но я хочу использовать хостлист домейнс, а с ним не работает, хотя с другими сайтами норм. Смотрел инструменты веб-разработчика (F12) в Firefox, вписывал от туда все домены что видел через запятую. В Wireshark включено показывать домены, также всё вписывал, но не помогло. Также без hostlist domains сайт в Wireshark выделен синим, а без хостлиста выделен темно-синим с красным текстом. cloudflare-ech.com, cloudflareinsights.com вписывал тоже.

Как я понял я просто не вписал какой-то домен, подскажите, как его узнать?

**Comment by numbereleven-a** [comment]
а что за сайт то, так то можно autohostlist попробовать

**Reply by bol-van** [reply]
Для начала убедиться, что нет стратегий нулевой фазы. syndata или wssize
Затем запустить tpws/nfqws с --debug и смотреть в консоли, либо --debug=@/tmp/debug.log

**Reply by icynstant** [reply]
> а что за сайт то, так то можно autohostlist попробовать

hotleaks.tv, уже пробовал. он вписал туда только cloudflare-ech.com и открылся.

**Reply by icynstant** [reply]
> Для начала убедиться, что нет стратегий нулевой фазы. syndata или wssize

юзаю первую tls1.3 с blockcheck'а, повторюсь, работает, пока не начну использовать --hostlist-domains=""

--wf-l3=ipv4 --wf-tcp=443 --dpi-desync=multidisorder --dpi-desync-split-pos=2

**Reply by bol-van** [reply]
Учимся пользоваться debug логом

**Comment by icynstant** [comment, marked as answer]
Помогло делать вайтлист айпишников вместо вайтлиста доменов, вроде как потому что 2 разных IP были привязаны к одному домену.
